packages/next/server/render.tsx

```diff
@@ -467,6 +486,12 @@ export async function renderToHTML(
     return ''
   }
 
+  let { renderServerComponentData } = renderOpts
```

Comment:
Does `renderServerComponentData` has to be extracted from `renderOpts`, or it could be a normal variable here `let renderServerComponentData = false`

Reply:
It has to be passed here for now. The current implementation mixes both edge and node runtimes, one parses this from the outside and one doesn’t. We will do the refactoring in follow-up PRs.